CDS_retriever.py

```diff
@@ -61,7 +69,7 @@ def year_retrieve(dataset, var, freq, year, grid, levelout, area, outdir, reques
         raise ValueError(f'Unknown dataset {dataset} requested')
 
     # extract time information
-    product_type, day, time, time_kind, minimum_steps = define_time(freq)
+    product_type, day, t, time_kind, minimum_steps = define_time(freq)
```

Comment:
where this t come from? you did not modify the `define_time` function here, correct?

Reply:
The 't' is used to avoid conflicts with the 'time' package, which I use later (line 150) to set a timer between successive calls to the download method of the CDS client.

Reply:
I did not modify ´define_time'

Reply:
sorry I misread, I understand now